pkg/action/git-credential.go

```diff
@@ -0,0 +1,189 @@
+package action
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/justwatchcom/gopass/pkg/store/sub"
```

Comment:
Missing gofmt?

Reply:
I used goimports, which handles it like that...